packages/interfaces/src/IAppliance.js

```diff
@@ -30,6 +30,18 @@ class IAppliance {
 		}
 	}
 
+	/**
+	 * Checks to ensure that third party dependencies are avialable to the appliance.
```

Comment:
```suggestion
	 * Checks to ensure that third party dependencies are available to the appliance.
```
You did this just for me, didn't you? 🤗